lockbox-ios/Store/AccountStore.swift

```diff
@@ -8,57 +8,65 @@ import RxCocoa
 import FxAClient
 import SwiftKeychainWrapper
 import WebKit
-
-enum KeychainKey: String {
-    // note: these additional keys are holdovers from the previous Lockbox-owned style of
-    // authentication
-    case email, displayName, avatarURL, accountJSON
-
-    static let allValues: [KeychainKey] = [.accountJSON, .email, .displayName, .avatarURL]
+import Shared
+
+/* These UserDefault keys are maintained separately from the `UserDefaultKey`
+ * enum in BaseConstants.swift because, for the time being, they are only
+ * useful in the main application. However, rather than maintaining two
+ * separate UserDefaults instances, all the values for these keys are stored
+ * in the app group instance so that no further migrations will be required
+ * in the case that they become relevant in an app extension context. */
+enum LocalUserDefaultKey: String {
```

Comment:
I'm not sure I understand the need for two sets of keys, especially if there is scope for sharing the keys.

Reply:
It has to do with minimizing what settings are included in the extension. eg, we don't need to share the preferred browser setting in the extension.